### PR TITLE
Feature/compound migration

### PR DESCRIPTION
**Pull request by mgnsharon**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+export WEB3_INFURA_PROJECT_ID=13cbb350551a4b0a8f383db6651de5dc
```
Comment (jeffywu) [inline .env:1]: @mgnsharon this should not be committed.

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,9 @@ __pycache__
 .hypothesis/
 build/
 reports/
-venv/
\ No newline at end of file
+venv/
```
Comment (jeffywu) [inline .gitignore:6]: artifacts from the hardhat compile should be added to .gitignore

File: scripts/inspect.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            (assetToken, underlyingToken) = notional.getCurrency(id)
+            at_token_address, _, _, at_token_type, _ = assetToken
+            ut_token_address, _, _, ut_token_type, _ = underlyingToken
+            print(f"Asset - {TokenTypeNames[at_token_type]} : {at_token_address}")
+            print(f"Under - {TokenTypeNames[ut_token_type]} : {ut_token_address}\n")
+
+    except ContractNotFound:
+        print(f"Contract not found at address: {addresses['notional']}")
+
+    # create env dictionary
```
Comment (jeffywu) [inline scripts/inspect.py:48]: Maybe remove these comments?

File: .gitignore (hunk 2 above)
Comment (mgnsharon) [reply .gitignore:6]: added artifacts/ and .env to gitignore, provided a sample .env

File: hardhat.config.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+require("@nomiclabs/hardhat-etherscan");
+
+module.exports = {
+  defaultNetwork: "hardhat",
+  networks: {
+    kovan: {
+      url:  "https://eth-kovan.alchemyapi.io/v2/H8E5dk64j-odPJlWF6TYnBYKjkZHYjuD"
```
Comment (jeffywu) [inline hardhat.config.js:7]: @mgnsharon this file should not be committed. Contains API keys

Comment (mgnsharon) [reply hardhat.config.js]: Moved the url's and api key to .env and added dotenv to hardhat config

**Pull request merged**